src/icons.h

```diff
@@ -63,6 +63,7 @@
 #define ICON_CPLUSPLUS     ICON_STR(MFIZZ_CPLUSPLUS, "", ICON_C)
 #define ICON_DATABASE      ICON_STR(MFIZZ_DATABASE_ALT2, "󰆼", "🗃️ ")
 #define ICON_DESKTOP       ICON_STR(FA_DESKTOP, "󰟀", "🖥️ ")
+#define ICON_DJVU          ICON_STR("FA_PAPERCLIP", "", "📎")
```

Comment:
The `FA_*` stuff are macros, you're not supposed to surround them in quotes.

```suggestion
#define ICON_DJVU          ICON_STR(FA_PAPERCLIP, "", "📎")
```